web/core/components/workspace/sidebar/favorites/favorites-menu.tsx

```
@@ -263,55 +241,34 @@ export const SidebarFavoritesMenu = observer(() => {
-                              handleDrop={handleDrop}
-                            />
-                          )}
-                        </Tooltip>
+                      {fav?.is_folder ? (
+                        <FavoriteFolder
+                          favorite={fav}
+                          isLastChild={index === length - 1}
+                          handleRemoveFromFavorites={handleRemoveFromFavorites}
+                          handleRemoveFromFavoritesFolder={handleRemoveFromFavoritesFolder}
+                          handleDrop={handleDrop}
+                        />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)